src/controllers/candidatesController.ts

```diff
@@ -0,0 +1,119 @@
+import candidatesService from '../services/candidatesService';
+
+import { validationResult } from 'express-validator';
+import ApiError from '../errors/ApiError';
+
+import CandidateInfo from '../types/CandidateInfo';
+import { NextFunction, Request, Response } from 'express';
+import Candidate from '../models/Candidate';
+
+class CandidatesController {
+    public async addCandidateInfo(
+        req: Request,
+        res: Response,
+        next: NextFunction,
+    ): Promise<CandidateInfo | void> {
+        try {
+            const errors = validationResult(req);
+            if (!errors.isEmpty()) {
+                return next(
+                    ApiError.BadRequest('Validation error', errors.array()),
+                );
+            }
+
+            const {
+                name,
+                surname,
+                email,
+                skype,
+                phone,
+                education,
+                technology,
+            } = req.body;
+
+            const cv = req?.files?.cv;
+            if (!cv) {
+                return next(ApiError.BadRequest('No file'));
+            }
+
+            const cvName = cv.name;
```

Comment:
What will be if 2 candidats will have same names of cv?
Why don't u add id, name, surname to cv name to make it more unique